src/LuauRenderer/nodes/expressions/renderStringLiteral.ts

```diff
@@ -25,7 +25,7 @@ function needsBracketSpacing(node: luau.StringLiteral) {
 
 export function renderStringLiteral(state: RenderState, node: luau.StringLiteral) {
 	const isMultiline = node.value.includes("\n");
-	if (!isMultiline && !node.value.includes('"')) {
+	if ((node.parent && luau.isInterpolatedString(node.parent)) || (!isMultiline && !node.value.includes('"'))) {
```

Comment:
This context-based not-quite-accurate return value is quite awkward. I think this might work better by not using `luau.StringLiteral` for the constant parts of `luau.InterpolatedString`, which aren't _really_ strings to begin with. It could just be a special node, like `MapField`.
We could also catch the StringLiteral before rendering it in renderInterpolatedString, I'll leave a comment there with a suggestion on what I mean.

Reply:
Having a separated node for the constant parts of the interpolated strings sounds way better. I think `luau.isField` and the `// fields` comments should be renamed to `luau.isSpecialNode` and `// special nodes` since the new node isn't exactly a "field" and currently the check isn't being used anywhere.